docs/startup/code-examples/versioned/index.md

```diff
@@ -47,3 +47,67 @@ In addition to the cookbook, you can also consult these:
 
 
 - [Amazon Kinesis](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/kinesis) and [Azure Eventhubs](https://github.com/apache/samza-hello-samza/tree/latest/src/main/java/samza/examples/azure) examples that cover how to consume input data from the respective systems.
+
+### Apache Beam API examples
+
+The easiest way to get a copy of the WordCount examples in Beam API is to use the following command:
+
+{% highlight bash %}
+> mvn archetype:generate \
+      -DarchetypeGroupId=org.apache.beam \
+      -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples \
+      -DarchetypeVersion=2.9.0 \
+      -DgroupId=org.example \
+      -DartifactId=word-count-beam \
+      -Dversion="0.1" \
+      -Dpackage=org.apache.beam.examples \
+      -DinteractiveMode=false
+{% endhighlight %}
+
+This command creates a maven project `word-count-beam` which contains a series of example pipelines that count words in text files:
+
+{% highlight bash %}
+> cd word-count-beam/
+
+> ls src/main/java/org/apache/beam/examples/
+DebuggingWordCount.java	WindowedWordCount.java	common
+MinimalWordCount.java	WordCount.java
+{% endhighlight %}
+
+To use SamzaRunner, please add the following `samza-runner` profile to `pom.xml`, same as in the [latest](https://github.com/apache/beam/blob/master/sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml). This change will also be available in the next Beam release.
```

Comment:
Suggest to be a bit more clear:   .... add the following samza-runner profile to pom.xml under the "profiles" section, same as in the ....

I also don't understand what "This change will also be available in the next Beam release." means when I first read it